src/components/rockets/rockets.js

```diff
@@ -0,0 +1,38 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import { fetchRockets, reserve } from '../../redux/rockets/rockets';
+import './rockets.css';
+
+function Rockets() {
+  const { rockets, status } = useSelector((state) => state.rockets);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    if (!status) dispatch(fetchRockets({ method: 'GET' }));
+  }, [status, dispatch]);
+
+  const reserves = (bool) => {
+    if (bool) return <span>reserved</span>;
+    return null;
+  };
+  return (
+    <section className="rockets">
+      {rockets.map((rocket) => (
+        <div key={rocket.id} className="rocket">
+          <img src={rocket.image} alt={rocket.name} />
+
+          <div className="rocket-info">
+            <h2>{rocket.name}</h2>
+            <p>
+              { reserves(rocket.reserved) }
```

Comment:
- [ ] Instead of showing the "Reserved" badge like this kindly use the `React conditional rendering syntax` as mentioned in the project requirements 

```javascript
{rocket.reserved && ( 
    // show the badge
)}
```

Reply:
Thank you, we will fix this